app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/MetadataEditingFragment.kt

```diff
@@ -25,13 +25,13 @@ import kotlinx.coroutines.launch
 import javax.inject.Inject
 
 @AndroidEntryPoint
-class CategoriesEditingFragment : Fragment() {
+class MetadataEditingFragment : Fragment() {
```

Comment:
I like this renaming, actually that would discard my comment above. But then let's also change the menu item's name accordingly.

Reply:
Forgot this one you're right ! I will update that :)